### PR TITLE
[zh-cn]: add the translation of SharedArrayBuffer.{grow(), growable}

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#17175

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/grow/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  - : 新的长度，以字节为单位，`SharedArrayBuffer` 调整后的大小。
+
+### 返回值
+
+无（{{jsxref("undefined")}}）。
+
+### 异常
+
+- {{jsxref("TypeError")}}
+  - : 如果当前 `SharedArrayBuffer` 是不可增大的的，则抛出该异常。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/grow/index.md:30]:
```suggestion
  - : 如果当前的 `SharedArrayBuffer` 不可增大，则抛出该异常。
```

File: files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/grow/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- {{jsxref("RangeError")}}
+  - : 如果 `newLength` 大于当前 `SharedArrayBuffer` 的 {{jsxref("SharedArrayBuffer/maxByteLength", "maxByteLength")}} 或小于 {{jsxref("SharedArrayBuffer/byteLength", "byteLength")}}，则抛出该异常。
+
+## 描述
+
+`grow()` 方将 `SharedArrayBuffer` 增大到 `newLength` 参数指定的大小，前提是 `SharedArrayBuffer` 是[可增大的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/growable)，并且新的大小小于等于当前 `SharedArrayBuffer` 的 {{jsxref("SharedArrayBuffer/maxByteLength", "maxByteLength")}}。新字节被初始化为 0。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/grow/index.md:36]: mdn/content#30912

```suggestion
`grow()` 方将 `SharedArrayBuffer` 增大到 `newLength` 参数指定的大小，前提是 `SharedArrayBuffer` 是[可调整大小的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/growable)，并且新的大小小于等于当前 `SharedArrayBuffer` 的 {{jsxref("SharedArrayBuffer/maxByteLength", "maxByteLength")}}。新字节被初始化为 0。
```

File: files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  sourceCommit: 16bacf2194dc9e9ff6ee5bcc65316547cf88a8d9
+---
+
+{{JSRef}}
+
+{{jsxref("SharedArrayBuffer")}} 实例的 **`growable`** 访问器属性返回该 `SharedArrayBuffer` 是否可增大。
+
+## 描述
+
+`growable` 属性是一个访问器属性，其设置访问器函数为 `undefined`，因此你只能读取该属性。该属性的值在构造数组时确定。如果在构造函数中 `maxByteLength` 选项被设置，`growable` 将返回 `true`；否则，它将返回 `false`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md:14]: Should we translate the term "set" here?

If so, we may need to align all the translation of "set" and "get".

/cc @jasonren0403

File: files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## 示例
+
+### 使用 growable
+
+在这个示例中，我们创建一个 8 字节缓冲区，该缓冲区可调整大小到的最大长度是 16 字节，然后检查其 {{jsxref("SharedArrayBuffer/growable", "growable")}} 属性，如果 `growable` 返回 `true`，则增大它：
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md:20]: mdn/content#30912

```suggestion
在这个示例中，我们创建一个 8 字节缓冲区，该缓冲区可调整大小到的最大长度是 16 字节，然后检查其 `growable` 属性，如果 `growable` 返回 `true`，则增大它：
```

File: files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/grow/index.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+### 异常
+
+- {{jsxref("TypeError")}}
+  - : 如果当前的 `SharedArrayBuffer` 不可增大，则抛出该异常。
+- {{jsxref("RangeError")}}
+  - : 如果 `newLength` 大于当前 `SharedArrayBuffer` 的 {{jsxref("SharedArrayBuffer/maxByteLength", "maxByteLength")}} 或小于 {{jsxref("SharedArrayBuffer/byteLength", "byteLength")}}，则抛出该异常。
+
+## 描述
+
+`grow()` 方将 `SharedArrayBuffer` 增大到 `newLength` 参数指定的大小，前提是 `SharedArrayBuffer` 是[可调整大小的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/growable)，并且新的大小小于等于当前 `SharedArrayBuffer` 的 {{jsxref("SharedArrayBuffer/maxByteLength", "maxByteLength")}}。新字节被初始化为 0。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/grow/index.md:36]:
```suggestion
`grow()` 方将 `SharedArrayBuffer` 增大到 `newLength` 参数指定的大小，前提是 `SharedArrayBuffer` 是[可增大的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/growable)，并且新的大小小于等于当前 `SharedArrayBuffer` 的 {{jsxref("SharedArrayBuffer/maxByteLength", "maxByteLength")}}。新字节被初始化为 0。
```

File: files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md (hunk 3 above)
Comment (jasonren0403) [reply files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md:14]: Just treat "set"(设置) and "get"(获取) as normal verb here?

If we encountered context like "setter" and "getter" I think we may leave them untranslated 🤔

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md:14]: Any updates?

Comment (yin1999) [reply files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md:14]: Okay. Let's translate it. I believe translating them won't cause ambiguity. Thank you all

File: files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/grow/index.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
+---
+title: SharedArrayBuffer.prototype.grow()
+slug: Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/grow
+l10n:
+  sourceCommit: 27180875516cc311342e74b596bfb589b7211e0c
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/grow/index.md:5]:
```suggestion
  sourceCommit: 509fde687ed349da8017a546f9cf094163f24844
```

File: files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+---
+title: SharedArrayBuffer.prototype.growable
+slug: Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/growable
+l10n:
+  sourceCommit: 16bacf2194dc9e9ff6ee5bcc65316547cf88a8d9
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/sharedarraybuffer/growable/index.md:5]:
```suggestion
  sourceCommit: 509fde687ed349da8017a546f9cf094163f24844
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**